scripts/airdrop.js

```diff
@@ -21,12 +23,56 @@ const argv = default_yargs
     describe: "Do not actually send transactions, just simulate their submission",
   }).argv
 
+const getFileExtension = function (filename) {
+  const parts = filename.split(".")
+  return parts[parts.length - 1].toLowerCase()
+}
+
+const parseCsvFile = async function (filePath) {
+  return new Promise((resolve, reject) => {
+    const results = []
+    fs.createReadStream(filePath, { encoding: "utf-8" })
+      .pipe(csv())
+      .on("data", (data) => results.push(data))
+      .on("end", () => resolve(results))
+      .on("error", (error) => reject(error))
+  })
+}
+// Note that this function is very specific to the incentive program itself
+// It uses gno_estimation for amount and hardcoded GNO_ADDRESS
+const toPayment = function (leaderBoardItem) {
+  const GNO_ADDRESS = "0x6810e776880C02933D47DB1b9fc05908e5386b96"
+  const { trader: receiver, gno_estimation: amount } = leaderBoardItem
+  return {
+    tokenAddress: GNO_ADDRESS,
+    receiver,
+    amount: new BigNumber(amount),
+  }
+}
+
+const parseTransferFile = async function (filename) {
+  const ext = getFileExtension(filename)
+
+  if (ext === "csv") {
+    const results = await parseCsvFile(filename)
+
+    const payments = results.map(toPayment).filter((payment) => !payment.amount.isZero())
+    return payments.map(({ amount, receiver, tokenAddress }) => ({
+      amount: amount.toString(10),
+      receiver,
+      tokenAddress,
+    }))
+  } else if (ext == "json") {
+    return JSON.parse(await fs.readFile(filename, "utf8"))
+  }
```

Comment:
nit: I think you should handle the extra case where its neither kind of file
```suggestion
  } else {
    throw new Error(`unsupported file type .${ext}`)
  }
```